src/platforms/android/configuration/integrations/logcat.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Logcat
+caseStyle: canonical
+supportLevel: production
+description: "Learn more about the Sentry Logcat integration for the Android SDK."
+categories:
+  - mobile
+---
+
+<Note>
+
+Supported in Sentry's Android SDK version `6.17.0` and above.
+
+Supported in Sentry Android Gradle Plugin version `3.5.0` and above.
+
+</Note>
+
+The Sentry Logcat integration adds support for automatically attaching breadcrumbs to events above a minimum logging level. Only explicit log calls are captured as breadcrumbs, logs that were created by any third party aren't captured.
+
```

Comment:
Could you add a screenshot similar as well here? Here's how I did it for Jetpack Compose: https://github.com/getsentry/sentry-docs/pull/6510/files